src/app/zap-templates/zcl/data-model/chip/sample-mei-cluster.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Sample Mei</name>
+    <!-- The MSB 16 bits of <code> are the VendorID. Replace this with your
+         VendorID. 0xFFF1 is a Test VendorID.
+
+         The LSB 16 bits of <code> are a self-assigned ClusterID -->
+    <code>0xFFF1FC20</code>
+    <define>SAMPLE_MEI_CLUSTER</define>
+    <description>The Sample MEI cluster showcases a cluster manufacturer extensions</description>
```

Comment:
Here's a question: Why is the Unit Testing cluster not a sufficient showcase for this?

Reply:
I think Unit Testing cluster, being quite exhaustive in its data model, has a different purpose than showcasing to a vendor how to implement custom functionality through manufacturer extensions.

I try here to create a minimally viable sample XML & source code to which we can point vendors interested in the topic

Reply:
OK.